src/payment/strategies/amazon-maxo/amazon-maxo-payment-initialize-options.ts

```diff
@@ -0,0 +1,3 @@
+export default interface AmazonMaxoPaymentInitializeOptions {
+    walletButton?: string;
```

Comment:
where is this being used? should this be `containerId` ?

Reply:
It  was not used yet so it was removed.